article/test/ArticleFeatureTest.scala

```diff
@@ -494,7 +494,6 @@ import collection.JavaConversions._
         findFirst("aside").getAttribute("role") should be("complementary")
         findFirst("header").getAttribute("role") should be("banner")
         findFirst(".l-footer__secondary").getAttribute("role") should be("contentinfo")
-        findFirst("nav").getAttribute("role") should be("navigation")
```

Comment:
When we say we don't support those browsers, it means we don't care if they look bad. They still have to usable. I think the guidance is that content needs to be readable/viewable and you need to be able to navigate. 

The navigation definitely needs to at least sort of work for browsers we don't support.

We say:
> We believe that our journalism should be accessible to everyone.

and 

> This means every feature of the website needs to consider how it adapts itself to each user's environment.

in our [Browser Support Principals](https://github.com/guardian/frontend/blob/master/docs/04-quality/02-browser-support-principles.md#browser-support-principles). But I think we should make this more clear and probably give examples of what this means in practise